src/main/scala/beam/router/osm/TollCalculator.scala

```diff
@@ -148,7 +148,8 @@ class TollCalculator @Inject()(val config: BeamConfig) extends LazyLogging {
       .flatMap(c => {
         c.split(" ")
           .headOption
-          .flatMap(token => Some(Toll(token.toDouble, Range("[:]"))))
+          .flatMap(token => try { Some(token.toDouble) } catch { case _: Throwable => None })
```

Comment:
Most likely not worth changing, but interesting to be aware of:

    import scala.util.control.Exception._
    ...
    .flatMap(token => catching(classOf[Throwable]).opt(token.toDouble))
    .map(token => Toll(token, Range("[:]"))))

Note that the last statement can just be a `map` since you're flattening the Option anyway.